Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -200,8 +183,14 @@ class BTHTTP: NSObject, URLSessionTaskDelegate {
             throw BTHTTPError.missingBaseURL(errorUserInfo)
         }
         
-        let mutableParameters = NSMutableDictionary(dictionary: parameters ?? [:])
-
+        var mutableParameters = NSMutableDictionary()
```

Comment:
We need to turn the encodable params into a dictionary to append the `authorization_fingerprint` below